code/modules/mining/satchel_ore_box.dm

```diff
@@ -45,7 +45,7 @@
 /obj/structure/ore_box/proc/dump_box_contents()
 	var/drop = drop_location()
 	var/turf/our_turf = get_turf(src)
-	for(var/obj/item/stack/ore/O in src)
+	for(var/obj/item/O in src)
 		if(QDELETED(O))
 			continue
 		if(QDELETED(src))
```

Comment:
is `QDELETED(src)` like nessassary? When could this happen? These checks are way too paranoid.

Reply:
These checks appear to be ancient, and in truth I'm of the mind to avoid increasing the scope of this 100+ file PR much further, but I think I logic here, since the ore box could be set on fire, burning plasma ore inside which could delete either the ore OR the box, which I could see happening at least once or twice.